code/__DEFINES/dcs/signals/atom/signals_movable.dm

```diff
@@ -11,6 +11,9 @@
 	#define COMPONENT_CANCEL_MOVE (1<<0)
 /// From /turf/open/gm/river/Entered(): (turf/open/gm/river/river, covered)
 #define COMSIG_MOVABLE_ENTERED_RIVER "movable_entered_river"
+/// From /atom/movable/proc/doMove: I think it only works with forceMove so watch out
```

Comment:
then make it wordier and specify that in the signal. COMSIG_MOVABLE_FORCEMOVE_PRE_CROSSED